src/components/Search/Search.js

```diff
@@ -72,6 +72,9 @@ const Search = React.createClass({
     }
     return null;
   },
+  getDropdownVisible: function () {
```

Comment:
I tried making these getters, but then `this` didn't have `props` or `state`... Not sure why?